components/prombench/manifests/results/alertmanager_config.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: alertmanager
+data:
+  config.yml: |-
+    global:
+      resolve_timeout: 5m
+
+    # The root route on which each incoming alert enters.
+    route:
+      group_by: ['namespace']
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 4h 
+
```

Comment:
These values are the default values in alertmanager.

Reply:
and how does this work with the `for: 2m` inside the rules ?